### ISSUE TITLE
Plugin for Game engine such as Unity

### ISSUE THREAD

**Issue by keli95566** [NVlabs/instant-ngp#710]
Does anyone have an idea how to use this amazing toolkit in the Unity game engine? Such that we could view real-time training results via openXR/steamVR.

**Comment by MS-2** [comment]
ward

**Comment by tap2k** [comment]
In interested in working on this

**Comment by Byronius7X** [comment]
I just discovered this really cool project from a few years ago called Colibri VR that uses colmap, so i input one of my outpupts from instant-ngp, it's a little wonky, but it seems like it's definitely possible. (i've never used unity before). Apparently the goal is to be able to put together a scene from real photographs (or video turned into photos in my case) and be able to move around in VR.

Interestingly, I discovered this while trying to figure out how to get instant-ngp and ffmpeg to render a second camera nudged slightly to the right(or left) for stereoscopic video, which would also be very cool.

**Comment by keli95566** [comment]
@Byronius7X  Thank you so much for sharing! This looks very promising! We were about to hack into the Cuda rendering pipeline and copy the video stream to Unity using Gstreamer 😅

**Comment by ffreality** [comment]
I am an Unreal Engine user but VDB export will work for both engine. We have vdb plugin. I think you guys also have that.

**Comment by keli95566** [comment]
@ffreality  thanks a lot for the response. Is there a VDB export option from instant-ngp? I see that testbed.exe is only responsible for training and rendering..

**Comment by ffreality** [comment]
@keli95566 nope, not currently but some users says there are some ways to implement it.
